src/wymeditor/editor/base.js

```diff
@@ -1037,9 +1037,12 @@ WYMeditor.editor.prototype.setRootContainer = function (sType) {
         firstNode,
         x;
 
-    if (sType.toLowerCase() === WYMeditor.TH) {
-        container = wym.selectedContainer();
+    container = wym.selectedContainer();
```

Comment:
nitpick. Could we call this variable `selectedContainer`? I know it was already called `container`, initially.